lib/fathead/opam/parse.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python
+
+import os
+from urlparse import urljoin
+
+from bs4 import BeautifulSoup
+
+__target__ = "https://opam.ocaml.org/"
+__module__ = "OPAM"
+
+class Environment(object):
+  """
+  The Environment class is responsible for setting up the directory
+  and retrieving the appropriate files from the web. It is made up of
+  a constructor and a 'setup/0' and 'get_contents/0' member functions.
```

Comment:
@pjhampton I think that in the official Python documentation is not common to see the format `method/arity`, instead, they tend to use `method()`, regardless of the number of arguments. wdyt?

BTW, maybe you're getting used to Elixir :+1: